apax/train/checkpoints.py

```diff
@@ -166,3 +166,29 @@ def restore_parameters(model_dir: Union[Path, List[Path]]) -> Tuple[Config, Froz
         )
 
     return config, params
+
+
+def canonicalize_energy_model_parameters(params):
+    """Ensures that parameters from EnergyDerivativeModels can be loaded
+    into EnergyModels by removing the "energy_model" parameter layer.
```

Comment:
They are definitely tossed out.
This is the structure of params for an EnergyDerivativeModel
```params = {
  "params": {
    "energy_model": {
      "atomistic_model": {
         ....
      }
  }
}
```

And this is the structure of params for an EnergyModel

```
params = {
  "params": {
    "atomistic_model": {
       ....
    }
}
```

keeping the params would amount to writing

`params = {"params": first_level}`

instead of 
`params = {"params": first_level["energy_model"]}`

Reply:
Ah ok now i see what your intention was. Perfect I think then it I ready to merge.